src/js/helpers/contractGrantActivityHelper.js

```diff
@@ -69,12 +69,12 @@ export const createSteppedAreaPath = (
     }, 'M')
 );
 /**
- * lineHelper
+ * validateLineValue
  * - determines if a line should be drawn
  * @param {Moment{}} date
  * @returns {null || Number}
  */
-export const lineHelper = (date, xDomain) => {
+export const validateLineValue = (date, xDomain) => {
```

Comment:
isn't this like `getLineValue`? It's returning a Number. `validate...` makes me think of something that returns a boolean

Reply:
Please let me know what this should be named and I will make it that. `validateAndGetLineValue`?

Reply:
```suggestion
export const getLineValue = (date, xDomain) => {
```